tyk-pro/values.yaml

```diff
@@ -76,6 +78,21 @@ mongo:
   # Default value: false
   useSSL: false
 
+postgres:
+  # Default value: tyk-postgres-postgresql.{{ .Release.Namespace }}.svc.cluster.local
+  # host: tyk-postgres-postgresql.tyk.svc.cluster.local
+  # Default value: 5432
+  # port: 5432
+  # Default value: postgres
+  # user: postgres
+  # password: 
+  # Default value: tyk_analytics
+  # database: tyk_analytics
+
+  # Enables SSL for MongoDB connection. MongoDB instance will have to support that.
+  # Default value: false
+  useSSL: false
```

Comment:
Is this vestigial? It references mongo in the comment, and I don't see a reference to `postgres.useSSL` anywhere. Would be nice to be able to set the sslmode for the connection string.

Reply:
Yeah sorry copy paste error 😆

Reply:
@drpebcak if I am not mistaken you are referring to the `sslmode` option on the connection string right?

Reply:
Yes, that's what I mean

Reply:
Makes perfect sense. I am thinking I should I just allow anyone to add whatever parameters to the postgres object in the values.yaml and then collapse that to create the connection string. That should give everyone the ability to do whatever they need to do.